internal/healthchecks/api_check.go

```diff
@@ -97,7 +97,7 @@ func runLoggingCheck(logger *log.Logger) error {
 	if err != nil {
 		return fmt.Errorf("can't get GCE metadata: %w", err)
```

Comment:
After thinking this after a couple of days. I think removing the `err` here is a good idea. We already have a `Network Check` that queries the metadata server and the error here might (in a super rare case) could expose PII details (e.g. can't find attribute `x`). The `API Check` responsibility is not to verify the metadata server.

Just leaving (or improving a little) the message should be enough. 
```
fmt.Errorf("can't get GCE metadata`)
``` 

Synced with @LujieDuan about this before. CC if he has more opinions about this.

Reply:
SGTM